pkg/apis/kubermatic/v1/user.go

```diff
@@ -88,6 +88,7 @@ type UserSettings struct {
 	CollapseSidenav            bool   `json:"collapseSidenav,omitempty"`
 	DisplayAllProjectsForAdmin bool   `json:"displayAllProjectsForAdmin,omitempty"`
 	LastSeenChangelogVersion   string `json:"lastSeenChangelogVersion,omitempty"`
+	UseClustersView            bool   `json:"useClustersView"`
```

Comment:
To maintain backward compatibility all new fields has to be:
* omitempty (i.e. optional)
* their lack should be handled in code (i.e. code should be OK with the fact that field wasn't set)